### PR TITLE
Feature/review

### PR DESCRIPTION
**Pull request by claire-lovisa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Append the following entry:
       }
     },
     callback: {
-      url: 'http://name/of/service/in/cache/graph/delta',
+      url: 'delta-producer-json-diff-file-publisher-leidinggevenden/delta',
```
Comment (cecemel) [inline README.md]: I think it is worth mentioning it should be http ?

File: app.js [hunk 2]
```diff
@@ -40,15 +40,15 @@ app.post('/delta', async function( req, res ) {
         }
         catch(e){
           console.error(`General error processing delta ${e}`);
-          await storeError(e);
+          await storeError(e.message);
```
Comment (cecemel) [inline app.js]: I think not all errors have something in message. 
 `try {  throw 'hello'} catch(e) {console.log(e); console.log(e.message)}`
Most of them will `str(e)` to something sensible. 

perhaps then go for `e.message || e`

Comment (cecemel) [reply app.js]: (yeah i know, i've done the same thing :-) but corrected afterwards

File: delta-cache.js [hunk 3]
```diff
@@ -36,8 +36,11 @@ export default class DeltaCache {
       this.cache = [];
 
       try {
+        const outputDirectory = `${SHARE_FOLDER}/${RELATIVE_FILE_PATH}`;
+        fs.mkdirSync(outputDirectory, { recursive: true });
```
Comment (cecemel) [inline delta-cache.js:40]: Good, catch, are we sure it does not throw exception when the directory already exists?

Comment (claire-lovisa) [reply delta-cache.js:40]: Yes thanks to the `{ recursive: true }` option, it will not throw any errors if the directory already exists !

**Comment by claire-lovisa** [comment]
All fixed, merging!

**Pull request merged**